packages/devextreme/js/ui/date_range_box/strategy/rangeCalendar.js

```diff
@@ -31,19 +28,12 @@ class RangeCalendarStrategy extends CalendarStrategy {
     }
 
     getFirstPopupElement() {
-        const $popupWrapper = this._getPopup().$wrapper();
-
-        const $todayButton = $popupWrapper.find(`.${TODAY_BUTTON_CLASS}`);
-
-        if($todayButton.length) {
-            return $todayButton;
-        }
-
-        return $popupWrapper.find(APPLY_BUTTON_SELECTOR);
+        return $(this._getPopup().getAllElements()[0]);
     }
 
     getLastPopupElement() {
-        return this._getPopup().$wrapper().find(CANCEL_BUTTON_SELECTOR);
+        const elements = this._getPopup().getAllElements();
+        return $(elements[elements.length - 1]);
```

Comment:
We can use modern syntax to make this action shorter
```suggestion
        return $(elements.at(-1));
```

Reply:
There is no `.at` in the elements because it is our inner `renderer`

Reply:
you mean $(elements), but 'elements' (with no $) is just an array, right?

Reply:
I mean when there is jquery, the `elements` is jquery object, when there is no jquery, as in the tests by default, it uses [renderer](https://github.com/DevExpress/DevExtreme/blob/23_2/packages/devextreme/js/core/renderer_base.js#L46)